src/server/event_bus.c

```
@@ -0,0 +1,168 @@
+        if (e->remove_count < OBC_EVENT_MAX_SUBS)
+            e->remove_pending[e->remove_count++] = fn;
+        return;
+    }
+
+    remove_sub(e, fn);
+}
+
+void obc_event_fire(const obc_engine_api_t *api, const char *event_name,
+                    int sender_slot, void *data)
+{
+    if (!event_name) return;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)